src/coordinate.typ

```diff
@@ -68,16 +68,22 @@
     (c.name, c.at("anchor", default: "default"))
   }
 
-  // Check if node is known
-  assert(
-    name in ctx.nodes,
-    message: strfmt("Unknown element '{}' in elements {}", name, repr(ctx.nodes.keys()))
-  )
+  // Two stage anchor lookup:
+  //   1: elements => <anchor>
+  //   2: current group => anchors => <anchor>
+  let anchor = if name in ctx.nodes {
+    let node = ctx.nodes.at(name)
+    (node.anchors)(anchor)
+  } else if name in ctx.groups.last().anchors {
+    ctx.groups.last().anchors.at(name)
```

Comment:
Not sure about this: This way, anchors added via `copy-anchors` can not overwrite element anchors.